src/envoy/http/backend_auth/README.md

```diff
@@ -22,7 +22,6 @@ This filter records statistics.
  missing a token needed for the request. Two possible causes: 1) the `jwt_audience` specified in the
  route entry perFilterConfig for this filter PerRouteFilerConfig is not in the `jwt_audience_list` in
  the FilterConfig. 2) fails to fetch ID token.
-- `denied_by_no_route`: Number of API Consumer requests that are denied due to not route configurated.
```

Comment:
Instead of deleting the stat, we can change it to `passed_through_by_no_route`. May be useful for API Gateway team. @toddbeckman-g